src/ctree-element-screen/ctree-segment-container.html

```diff
@@ -187,16 +187,20 @@
 
       previous() {
         if (this._variationIndex <= 0) return;
-
+        this.$.component.editing = false;
         this._variationIndex--;
       }
 
       next() {
         if (!this.segment || !this.segment.variations || this._variationIndex >= this.segment.variations.length - 1) return;
-
+        this.$.component.editing = false;
         this._variationIndex++;
       }
 
+      getComponent() {
+        return this.shadowRoot.querySelector("#component");
```

Comment:
If you're updating references to ` this.shadowRoot.querySelector("#component")` I suggest doing a find and replace.  It's always a good practice to change similar things when possible (less time investment for improving the code).